music21/musicxml/m21ToXml.py

```diff
@@ -2529,7 +2541,10 @@ def instrumentSetup(self):
         {4.0} <music21.instrument.BassClarinet 'Bass clarinet'>
         '''
         # get a default instrument if not assigned
-        self.instrumentStream = self.stream.getInstruments(returnDefault=True, recurse=True)
+        if self.parent is not None and id(self.stream) in self.parent.instrumentsByStream:
+            self.instrumentStream = self.parent.instrumentsByStream[id(self.stream)]
```

Comment:
when would this take place?

Reply:
ah -- I get it from below.  Maybe a Note.  "Will be True if this is the second or subsequent PartStaff in a Group"  -- help people who are reading the code for a general case read it.